### DISCUSSION TITLE
False error: Expected corresponding JSX closing tag

### DISCUSSION THREAD

**Discussion by bradwoods** [mdx-js/mdx#2514] [Q&A]
I'm using mdx in next.js. I'm getting a false error reported:
`Expected corresponding JSX closing tag for '_components.LayoutColumn' ts-plugin(17002)`.

![image](https://github.com/user-attachments/assets/49852e4d-067d-401e-83e7-6f0e88e1a7ce)

Does anyone know a workaround for this or a way to ignore?

**Comment by remcohaszing** [comment]
This looks like a bug. Can you share the content to reproduce this as text?

**Reply by remcohaszing** [reply]
https://github.com/mdx-js/mdx-analyzer/issues/485
